shared/common-adapters/plain-input.d.ts

```diff
@@ -146,7 +146,7 @@ export type TextInfo = {
 export type InternalProps = {} & DefaultProps & Props
 
 declare class PlainInput extends React.Component<Props> {
-  defaultProps: DefaultProps
+  static defaultProps: DefaultProps
```

Comment:
what's this do?

Reply:
It's more correct, and makes this pass type checking
https://github.com/keybase/client/blob/9781138fe2bd2abce7d980a9727872c34f7183e7/shared/common-adapters/new-input.tsx#L77